setup.py

```diff
@@ -3,10 +3,12 @@
 
 from setuptools import setup
 
+import bravado_asyncio
```

Comment:
I would not suggest to import your module here ... I remember that it could create issues while installing the package (not sure about the exact error).
Different workarounds are available (ie. https://github.com/striglia/pyramid_swagger/blob/master/setup.py#L11) 

Feel free to drop this ... it's possible that my memory is wrong 😃

Reply:
It's what we do in bravado, but I remember posts about potential issues as well... I'll change it. :)